scripts/setup-k8s-master.sh.in

```diff
@@ -56,6 +60,8 @@ EOF
 kubeadm init --config /tmp/kubeadm.yaml
 rm /tmp/kubeadm.yaml
 
+kubectl --kubeconfig=/etc/kubernetes/admin.conf get cm -n kube-public cluster-info -o json | jq -r .data.kubeconfig | aws s3 cp - s3://{{ClusterInfoBucket}}/cluster-info.yaml
```

Comment:
`kubectl` has a built-in jsonpath implementation, so I bet we could replace `jq` here with `-o jsonpath='{.data.kubeconfig}'. `

That way we can remove that extra dependency.

Reply:
👍

Reply:
nice, yeah good call